src/powertools/utilities.ts

```diff
@@ -0,0 +1,25 @@
+import { Logger } from '@aws-lambda-powertools/logger';
+import { Metrics } from '@aws-lambda-powertools/metrics';
+import { Tracer } from '@aws-lambda-powertools/tracer';
+
+const logger = new Logger({
+    persistentLogAttributes: {
+        awsAccountId: process.env.AWS_ACCOUNT_ID || 'N/A',
+        awsRegion: process.env.AWS_REGION
```

Comment:
will add the || 'N/A'

Reply:
Although that value is set by the AWS Lambda runtime and it's guaranteed to exist. But in terms of good practice in how to use PT I agree, better to add it.